lib/checks/shared/avoid-inline-spacing.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "avoid-inline-spacing",
+	"evaluate": "avoid-inline-spacing.js",
+	"metadata": {
+		"impact": "serious",
+		"messages": {
+			"pass": "No inline styles that affect text spacing has been specified",
+			"fail": "Inline styles that affect text spacing has been specified"
```

Comment:
You'll need to include what style props are failing. Something like:

> Remove `!important` from inline styles {= data =}, as overriding this not supported by most browsers

Reply:
I don't think `as overriding this not supported by most browsers` reads right here. I have done the below.

`Remove '!important' from inline style{{=it.data && it.data.length > 1 ? 's' : ''}} {{=it.data.join(', ')}} that affect text spacing`